src/solve/client/client.ts

```diff
@@ -0,0 +1,46 @@
+import https from "https";
+import axios, { AxiosInstance } from "axios";
+import { Chain } from "../types";
+import { FungibleService } from "./fungible";
+
+const API_URL = process.env.NEXT_PUBLIC_API_URL ?? "https://api.skip.money/v1";
+
+export const ENDPOINTS = {
+  GET_CHAINS: `${API_URL}/info/chains`,
+  GET_ROUTE: `${API_URL}/fungible/route`,
+};
+
+export const IGNORE_CHAINS = ["agoric", "8ball", "akashnet-2"];
```

Comment:
Should these be passed into the client as arguments?

Reply:
Yeah probably, updated